components/form-builder/panel/Modal.tsx

```diff
@@ -3,7 +3,8 @@ import styled from "styled-components";
 import { useTranslation } from "next-i18next";
 import PropTypes from "prop-types";
 
-import { Button, FancyButton } from "./Button";
+import { Button as OldButton } from "./Button";
```

Comment:
Will this get swapped at some point ?

Reply:
Yes, I will do it later in this PR. I am going to remove that other "Button" file entirely.